helix-core/src/test/java/org/apache/helix/cloud/event/TestCloudEventCallbackProperty.java

```diff
@@ -188,7 +193,7 @@ public void testUsingInvalidImplClassName() throws Exception {
     _helixManager.connect();
 
     // Manually trigger event
-    CloudEventHandlerFactory.getInstance()
+    ((CloudEventHandler) CloudEventHandlerFactory.getInstance(CloudEventHandler.class.getCanonicalName()))
```

Comment:
can we add negative case where event handle is not registered and we call unregister etc

Reply:
When event handle is not registered then unregister should be a no-op. It is valid to add a test case but it probably won't be a negative test case.